scripts/copy_from_upstream/copy_from_upstream.yml

```diff
@@ -61,6 +61,13 @@ upstreams:
     sig_meta_path: 'META/{pretty_name_full}_META.yml'
     sig_scheme_path: '.'
     patches: [pqmayo-aes.patch, pqmayo-mem.patch]
+  -
+    name: upcross
+    git_url: https://github.com/rtjk/CROSS-PQClean.git
```

Comment:
Nicely done enabling use of our upstream integration logic! Do you intend to eventually contribute this to PQClean itself, too (i.e., change this link to the PQClean repo), adding some more testing and maintenance by the team there? What statement regarding maintenance of Cross can you give? Would you agree becoming a [liboqs contributor and codeowner for Cross](https://github.com/open-quantum-safe/liboqs/blob/main/GOVERNANCE.md#contributors)? If so, please indicate this by adding yourself to the CONTRIBUTORS file in an updated PR. Please also add the commit tag/message "[trigger downstream]" such as to trigger testing within oqs-provider.

Reply:
The team is definitely considering opening a PR to PQClean as well. We'll make sure to update the upstream if/when this happens.

Regarding maintenance: I'm working on this integration for my master thesis, I'd like to also add my advisors to the `CONTRIBUTORS` file. You can see them as authors in CROSS' header comments and they would be the main point of reference for any future development. Is [this](https://github.com/rtjk/liboqs/commit/110cd4204c25edbe62e3c590683f610f635ab65b) ok @baentsch?

Reply:
If those advisors are not really (continued/committed) contributors in the sense of OQS ([regularly willing to do PRs as well as review those within OQS](https://github.com/open-quantum-safe/liboqs/blob/main/GOVERNANCE.md#contributors)) (?) it'd indeed be best to add them (like yourself, given you're probably not continuing to work on this after your thesis (or are you?)) by reference only to the CONTRIBUTORS file (probably best with email address in case one needs to reach out privately, e.g., in case of security problems) but not to CODEOWNERS (implying a longer-term commitment): OK?

Reply:
Just in case the question above didn't hit your Inbox, tagging @rtjk explicitly on this (to me final for this PR) open question.

Reply:
Apologies for the delay @baentsch, I'm waiting for feedback from the team on this, I will get back to you soon and update codeowners/committers here and in oqs-provider. Thanks for your patience!

Reply:
@alexrow from the CROSS team, who actively maintains the [main repo](https://github.com/CROSS-signature/CROSS-implementation), has assisted me throughout this integration and is willing to oversee CROSS' evolution in liboqs, review pull reuqests, and manage the OIDs in oqs-provider. I just added him as codeowner for `copy_from_upstream.yml` and the CROSS subdirectory.

Reply:
Thanks for this commitment and welcome @alexrow! The upstream reference I see though, is to a repo owned by @rtjk : Is that something that should be changed then in a future PR or before Cross is integrated to a `liboqs` release (happening very soon)?

Reply:
Yes,

- [Repo 1](https://github.com/CROSS-signature/CROSS-implementation) hosts the unaltered NIST submission package.

- [Repo 2](https://github.com/rtjk/CROSS-PQClean) hosts the lboqs upstream, with all the changes that were necessary (e.g. switching to fixed sized integer types), as well as the code generation script that produces the 18 subdirectories (one for each parameter set) and applyes namespacing.

we might merge the two in the future, in which case we'll make sure to open a new PR to cahnge the upstream. Tagging @alexrow to confirm this.

Reply:
I do confirm that we're planning a restructuration of the repositories owned by the CROSS github organization; the expected plan is that, we will move Repo 2 under its ownership, while keeping the structure, so that it can act as an upstream for liboqs.

I also confirm that, once the state of the repos is changed, we'll submit a PR to change the upstream.

Thanks for all the work on liboqs!